dev-support/ci/selective_ci_checks.bats

```diff
@@ -195,7 +195,7 @@ load bats-assert/load.bash
 #@test "native test in other module" {
 #  run dev-support/ci/selective_ci_checks.sh 7d01cc14a6
 #
-#  assert_output -p 'basic-checks=["rat","author","checkstyle","findbugs","native","unit"]'
+#  assert_output -p 'basic-checks=["rat","author","checkstyle","findbugs","native"]'
```

Comment:
This is commented out but the `assert_output -p needs-integration-tests=false` stayed the same leaving it in an inconsistent state. Are there any reason to reenable this in the future?